src/components/Button.js

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+
+const Button = styled.button`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 335px;
+  height: 50px;
+
+  margin: 0 0 16px 0;
+  color: ${({ color }) => (color ? color : '#fffff')};
+  background-color: ${({ bgColor }) => (bgColor ? bgColor : 'transparent')};
```

Comment:
```suggestion
  color: ${({ color }) => color || '#fffff'};
  background-color: ${({ bgColor = 'transparent' }) => bgColor};
```

값이 없을 경우 기본 값을 주고 싶은 거라면 이런 식으로 쓸 수도 있겠어요.
기존 방법이든, 위의 두가지 방법이든 큰 차이는 없으니 한 번씩 확인해보세요! ☺️

Reply:
오오.. 감사합니다 !! 👀